packages/financial-templates-lib/test/price-feed/CreatePriceFeed.js

```diff
@@ -323,7 +323,7 @@ contract("CreatePriceFeed.js", function(accounts) {
   it("Valid CryptoWatch config", async function() {
     const config = {
       type: "cryptowatch",
-      apiKey,
+      cryptowatchApiKey: apiKey,
```

Comment:
Are you worried that this change (and the others for existing feeds in this PR) will break any 3rd party production bots? If we don't want to do a soft deprecation (i.e. take `apiKey`, but prefer `cryptowatchApiKey` if provided), then we should probably change the title to (notice the `!`):
```
refactor(create-price-feeds)!: Each PriceFeed renames apiKey config param
```

This will trigger a major version bump to denote the breaking change.

Reply:
Good point!

Reply:
This is a really neat feature that it triggers a major version bump